core/server/worker/src/test/java/alluxio/worker/block/BlockWorkerMetricsTest.java

```diff
@@ -52,19 +52,23 @@ public void before() throws Exception {
   }
 
   @Test
-  public void testMetricsCapacity() {
+  public void testMetricsCapacity() throws InterruptedException {
     when(mBlockStoreMeta.getCapacityBytes()).thenReturn(1000L);
     Assert.assertEquals(1000L, getGauge(MetricKey.WORKER_CAPACITY_TOTAL.getName()));
     when(mBlockStoreMeta.getUsedBytes()).thenReturn(200L);
+    // sleep 5 seconds because the timeout of this registered is CacheGauge,
+    // and it's update interval is 5 seconds
+    Thread.sleep(5000);
```

Comment:
Make this a constant in DefaultBlockWorker and reference it here from the tests.